client/extensions/woocommerce/app/order/order-details/row-total.js

```diff
@@ -2,45 +2,79 @@
 /**
  * External dependencies
  */
-import { localize } from 'i18n-calypso';
-import PropTypes from 'prop-types';
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { kebabCase, noop } from 'lodash';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import formatCurrency from 'lib/format-currency';
-import { getOrderTotalTax } from 'woocommerce/lib/order-values';
+import { getCurrencyDefaults } from 'lib/format-currency';
+import PriceInput from 'woocommerce/components/price-input';
 
 class OrderTotalRow extends Component {
 	static propTypes = {
-		order: PropTypes.shape( {
-			currency: PropTypes.string.isRequired,
-			total: PropTypes.string.isRequired,
-		} ),
+		currency: PropTypes.string.isRequired,
+		isEditable: PropTypes.bool,
+		label: PropTypes.string.isRequired,
+		name: PropTypes.string,
+		numberFormat: PropTypes.func,
+		onChange: PropTypes.func,
 		showTax: PropTypes.bool,
+		taxValue: PropTypes.oneOfType( [ PropTypes.number, PropTypes.string ] ),
+		translate: PropTypes.func.isRequired,
+		value: PropTypes.oneOfType( [ PropTypes.number, PropTypes.string ] ).isRequired,
+	};
+
+	static defaultProps = {
+		currency: 'USD',
+		isEditable: false,
+		onChange: noop,
+	};
+
+	renderEditable = () => {
+		const { className, currency, label, value, onChange, numberFormat } = this.props;
+		let name = this.props.name;
+		if ( ! name ) {
+			name = kebabCase( label );
```

Comment:
I'm not sure this is going to play well with https://github.com/Automattic/wp-calypso/pull/18633/files#diff-5c5d84013fcda316f077eff7c4e88b8dR89 which is expecting the name to either be `shipping_total` or in the format `something-#`

Reply:
It's a default, in case `name` isn't passed in -- `shipping_total` is passed in though. I suppose I could switch to `snakeCase` instead. The quantity inputs don't use this component.

Reply:
Ah - I see what you mean. None of the instances of OrderTotalRow get that onChange passed to it other than the shipping_total one. I leave it up to you if you want to switch to snake_case. Approving the PR.